src/ambassadors/images.ts

```diff
@@ -329,21 +367,25 @@ export const getAmbassadorImages = ((ambassador: AmbassadorKey | string) => {
 }) as ((ambassador: AmbassadorKey) => AmbassadorImages) &
   ((ambassador: string) => AmbassadorImages | undefined);
 
-// TODO: Can we derive this from ambassadors[].plush?
-type AmbassadorMerch = Partial<{
-  [key in AmbassadorKey]: AmbassadorImage;
-}>;
+type AmbassadorMerch = {
+  [key in AmbassadorWithPlushKey]: AmbassadorImage;
+};
 
 const ambassadorMerchImages: AmbassadorMerch = {
   stompy: { src: stompyImageMerch, alt: "" },
   georgie: { src: georgieImageMerch, alt: "" },
   winnieTheMoo: { src: winnieImageMerch, alt: "" },
 };
 
+export const isAmbassadorWithPlushKey = (
+  str: string
+): str is AmbassadorWithPlushKey =>
+  ambassadorKeys.includes(str as AmbassadorKey);
+
 export const getAmbassadorMerchImage = (
-  ambassador: AmbassadorKey | string
+  ambassador: AmbassadorWithPlushKey | string
 ): AmbassadorImage | undefined => {
-  if (!isAmbassadorKey(ambassador)) return undefined;
+  if (!isAmbassadorWithPlushKey(ambassador)) return undefined;
 
   return ambassadorMerchImages[ambassador];
 };
```

Comment:
This can probably use the same type overload style as `getAmbassadorImages` now, to guarantee a return for the type?

Reply:
can do